practical/DeepClustering/FlorianKittel/kmeans.py

```diff
@@ -0,0 +1,64 @@
+import torch
+
+
```

Comment:
I personally find it useful if there input and return  parameters of functions are described shortly, e.g. which shape can we expect for a given Tensor. But maybe that's a bad habit of me. But I find your code very good structured and clear.